### DISCUSSION TITLE
[求助]我不小心走到一个有点奇怪的地方

### DISCUSSION THREAD

**Discussion by Tragedysaver1024** [share121/inter-knot#572] [其他]
如题。
简而言之就是我今天走在六分街的路上，然后脚下一个不小心踩空了，就进到了如图所示这个地方……
有人能来救我一下吗？这里闪光灯发出的声音简直要让我抓狂了。
[![pAStEVK.jpg](https://s21.ax1x.com/2024/08/10/pAStEVK.jpg)](https://imgse.com/i/pAStEVK)

**Comment by gup444555** [comment]
丸辣

**Comment by 75020-maker** [comment]
丸辣

**Comment by GUILAISHIXV** [comment]
这个委托我接...接不了

**Comment by langxmo** [comment]
这个委托看着就难度很大

**Comment by ling75431** [comment]
丸辣

**Comment by Nirvana341** [comment]
丸辣

**Comment by March-777th** [comment]
这边建议您日墙卡bug试试呢

**Comment by Tragedysaver1024** [comment]
我还真是挺幸运的，走着走着就走到一个类似于档案馆的地方了……
里面好像有几篇文档，是一个叫M.E.G.的组织写的，里面还说离开这个鬼地方的唯一办法就是一头撞墙上……你们说我该咋整？

**Comment by npldqt** [comment]
你进入了零号空洞吧

**Comment by Tragedysaver1024** [comment]
没人回我我就只能自己壮胆子试试了，然后撞完墙之后我就来到一个类似于停车场的地方……
我还找到了几瓶饮料……杏仁味儿的，喝起来还不错，哈哈。

**Reply by xuan8767** [reply]
杏…杏仁味的？！

**Comment by StarrySkyehk** [comment]
这个委托我接...接不了

**Comment by yagamisakamijou** [comment]
会为您祈祷的

**Comment by WRYYY-S** [comment]
丸辣

**Comment by LINHUAIWAN** [comment]
祝您幸福

**Comment by Tragedysaver1024** [comment]
总而言之言而总之算是找到M.E.G.了，他们许诺给我的报酬应该是还不错，每个月40升杏仁水，只不过我想不清楚他们不发钱给我发这个做什么……

**Comment by zhangyj0523** [comment]
没救了

**Comment by Tragedysaver1024** [comment]
回上面几楼，我现在过得还算不错，现在正在那个破停车场（好像叫Level 1）里面的一个小餐馆吃饭……别说味道还行，老板也挺友善的

**Comment by YTYX-Github** [comment]
这个委托···我不接！

**Comment by caizibook** [comment]
丸辣！自求多福叭！

**Comment by lbghhj** [comment]
这个我就去过！
是空洞内的隐藏空间，没想到已经侵蚀到六分街了啊
这个只能用来自世界之外的力量才行！
那 就 是 左上角脱离卡死！

**Comment by hzjwork** [comment]
快去天花板上找涂鸦（

**Comment by Tragedysaver1024** [comment]
> 快去天花板上找涂鸦（

MEG的人警告我别这么做，这样做的人基本上就没几个能回来，能回来的也都变成怪物了

**Comment by Tragedysaver1024** [comment]
发个求助帖但是回复的只有丸辣，这也是末法之世的一个侧面啊（悲）

**Comment by Sushizhu** [comment]
这个委托我接了！

**Comment by ZMJ2333** [comment]
这个委托我接了（还有如果您还活着的话尽量前往一个类似于游戏厅的地方，在第777号游戏机那等我，会有一个黑白相间的邦布，耳朵上订了一个螺丝）（在这里不要轻易相信任何生物！！！这里不是好地方，它比零号空洞还危险！！！）
建议你找个安全的地方把这个看看：[后室指南lv.0~99](https://www.bilibili.com/video/BV1uj411C7JC/?spm_id_from=333.337.search-card.all.click&vd_source=bdad6febb499433cdecb6a42dc2b6261)

**Comment by su2569** [comment]
丸辣

**Comment by dominic121-dot** [comment]
丸辣

**Comment by qbsozo** [comment]
這個委託我就先不接了

**Comment by qiuledadada** [comment]
等待后续不知道楼主成功回来了没有但愿平安无事

**Comment by tiancaitu** [comment]
这个委托我接，，诶？告辞

**Comment by keT-6** [comment]
这个委托我接！！！？再见

**Comment by sishui-buyin** [comment]
楼主还在吗 出来了吗

**Comment by Gmadelyn7** [comment]
楼楼————一定要活着出来啊

**Comment by WORLD-executeME** [comment]
走好

**Comment by zifeiyu1111** [comment]
没见过的空洞类型，小心以骸

**Comment by 13579yangh** [comment]
哇超！你一定要活着呀！

**Comment by 123-Invincibleswordsman** [comment]
丸辣

**Comment by Jiucheng258** [comment]
这是什么类型的空洞？看着不太对劲啊，对了，你检测了那里的以太活性没

**Comment by sdsd122** [comment]
丸辣

**Comment by ZMJ2333** [comment]
> 这个委托我接了（还有如果您还活着的话尽量前往一个类似于游戏厅的地方，在第777号游戏机那等我，会有一个黑白相间的邦布，耳朵上订了一个螺丝）（在这里不要轻易相信任何生物！！！ 这里不是好地方，它比零号空洞还危险！！！ ） 建议你找个安全的地方把这个看看：[后室指南lv.0~99](https://www.bilibili.com/video/BV1uj411C7JC/?spm_id_from=333.337.search-card.all.click&vd_source=bdad6febb499433cdecb6a42dc2b6261)

@Tragedysaver1024 我的邦布到现场了，没看到您人啊，游戏厅那一层这么难进的么？算了，我等着吧（第一次知道后室里可以联网诶！之前都没试过带无线收发机进去的）

**Reply by dolokong** [reply]
兄啊，你的邦布大抵是出不来力（悲）

**Reply by Tragedysaver1024** [reply]
还有游戏厅这地方？我现在人在M.E.G.嘴里说的Level 1，别说过得还行

**Comment by GanlinSama** [comment]
欸我去，一整天了，楼主人呢？？？

**Comment by ZMJ2333** [comment]
不是哥们等了半天您人呢？虽然以太能可以续航很久，但也经不住这样耗啊我通过游戏厅去了M.E.G.的几个基地没看到您人啊…哥们回个话！这后室不是能联网么

**Comment by GanlinSama** [comment]
楼上那位，撑不住先撤吧，这儿也不知道是不是还在空洞里，万一呆的久了被侵蚀了可不好了
发个坐标给我我去接班吧，如果能把萝卜也借用一下就更好了嘿嘿(●ˇ∀ˇ●)

**Comment by ZMJ2333** [comment]
回楼上 @GanlinSama 这个地方的性质和空洞不一样，是没有侵蚀的，但是这里的环境会让人陷入疯狂（如果不呆在光亮的地方的话）。我还好，因为我是用邦布进去的

**Comment by natihawi** [comment]
楼主保重，实在不行撞墙也不失为一种办法(

**Comment by ARequest** [comment]
哥们只能自求多福了，这地方盗洞客都不敢进。

**Comment by Sui-Qing** [comment]
自求多福吧哥们qwq

**Comment by blqdswkl** [comment]
邦布进去了，找不到你啊

**Comment by RETRN1** [comment]
丸

**Comment by lllliufeng** [comment]
不会到其他层了吧

**Comment by WorldZHT** [comment]
送进去的邦布失联了！

**Comment by kjiko** [comment]
法厄同呢，救救楼主！！！

**Comment by su2569** [comment]
6



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年08月12日 11:32 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] [求助]我不小心走到一个有点奇怪的地方 (Discussion #572) |

法厄同呢，救救楼主！！！

**Comment by AiKun1145** [comment]
请你发一下你的WiFi热点

**Comment by Zhengjingren12138** [comment]
你那里居然有网

**Comment by cooperatior** [comment]
太绝望了我使用败者食尘......你根本不在杜王町你到底在哪。啊原来不在一个世界吗。

**Reply by ZMJ2333** [reply]
原来这里也有JO厨（替身在后室里是无敌的！雀信）

**Comment by ZMJ2333** [comment]
@blqdswkl 您的邦布还在后室里吗？我还在，您在哪一层？ @WorldZHT 您的邦布在哪里失联的，我看看能不能帮您找一下
@Tragedysaver1024 希望您已经逃离后室，反正我好像要废一个邦布了……，原本想随便探索一下，不小心被抓到 **level fun** 力(悲

**Comment by jianruiyu** [comment]
这个委托我接了

**Comment by RS64671012** [comment]
> @blqdswkl 您的邦布还在后室里吗？我还在，您在哪一层？ @WorldZHT 您的邦布在哪里失联的，我看看能不能帮您找一下 @Tragedysaver1024 希望您已经逃离后室，反正我好像要废一个邦布了……，原本想随便探索一下，不小心被抓到 **level fun** 力(悲

这下确定了，是Fandom版本
@Tragedysaver1024 [官网入口](https://backrooms.fandom.com/zh/wiki/Backrooms_Wiki)

**Comment by qiuledadada** [comment]
快两天了楼主还好吗？

**Comment by jinuo12345** [comment]
你到处找找看，看看可不可以找到去往其他层级的出口

**Comment by AJIU223** [comment]
丸啦

**Comment by Merlin-cyq** [comment]
哥们你废了，自求多福吧

**Comment by Phaethon-0019** [comment]
这个委托我……真没法接

**Comment by XiaoDao7755** [comment]
丸辣

**Comment by qingyangprince** [comment]
https://backrooms-wiki-cn.wikidot.com/进里面看看，注意安全

**Comment by youximi** [comment]
丸辣 这个委托我还真接不了 不过我可以给你后室攻略 祝你顺利
<iframe src="//player.bilibili.com/player.html?isOutside=true&aid=448734103&bvid=BV1uj411C7JC&cid=1273425805&p=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"></iframe>

**Comment by hdaihdsu** [comment]
我等级不够，我走了，哈哈，再见好自为之。

**Comment by laoyee234** [comment]
这个委托接不了

**Comment by 1451374815** [comment]
这个委托我接了

**Comment by 1451374815** [comment]
这个委托我接了，我在leave131，马上来接你

**Comment by lamtt26** [comment]
这是那个恐怖游戏吧

**Comment by hyj277** [comment]
新空洞？待会我去看看

**Comment by qiuledadada** [comment]
回复楼上这地方最好不要随便去，另外三天了楼主还没出来吗？祝好运

**Comment by tommy-4ever** [comment]
祝你好运

**Comment by ranrenknmv** [comment]
你可以四处逛逛，找一些杏仁水喝，当你遇到那些黑色的生物时，请为了你的生命逃跑

**Reply by dolokong** [reply]
> 你可以四处逛逛，找一些杏仁水喝，当你遇到那些黑色的生物时，请为了你的生命逃跑

别去，它们（实体）银多

**Comment by dolokong** [comment]
你大抵是出不来了，这地方比零号空洞还麻烦，不过好在后室有网，尽量保持和外界联系吧，别忘了去找M.E.G，他们能给你最大程度上的帮助。

**Reply by dolokong** [reply]
test

**Comment by dolokong** [comment]
> 没人回我我就只能自己壮胆子试试了，然后撞完墙之后我就来到一个类似于停车场的地方…… 我还找到了几瓶饮料……杏仁味儿的，喝起来还不错，哈哈。
 @Tragedysaver1024  小心液体痛苦！

**Reply by Tragedysaver1024** [reply]
没事，我刚查过数据库了，我喝那玩意儿不是红的应该没事

**Comment by Jie-Ket** [comment]
丸辣！

**Comment by huoxu12345** [comment]
这个委托我放弃了，加油（dog）

**Comment by SENMAOJU** [comment]
我不敢接

**Comment by YING-ADX** [comment]
丸辣

**Comment by adrianlao647** [comment]
救不了一点

**Comment by lyx1662** [comment]
丸辣

**Comment by MenaeM-Meow** [comment]
丸辣

**Comment by Cherss111** [comment]
丸辣！

**Comment by wuxiafeiguang** [comment]
这个委托.......丸辣！<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Comment by HAZDSB** [comment]
丸辣！

**Comment by qiang-di** [comment]
丸辣！

**Comment by qiang-di** [comment]
完了<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Comment by lhyxyz** [comment]
这个委托我。。。。。。接不了

**Comment by miedi-zhifu76** [comment]
这个委托我接了！

**Reply by miedi-zhifu76** [reply]
等等，解不了

**Reply by miedi-zhifu76** [reply]
呸，接不了

**Comment by zm0214** [comment]
丸辣😭😭😭

**Comment by ikungei** [comment]
这边建议您紫砂

**Comment by 1luik** [comment]
https://haokan.baidu.com/v?pd=wisenatural&vid=10047936241728117617

**Comment by Tragedysaver1024** [comment]
没人看一下我上面发的回复吗，耍我啊混账

**Comment by Tragedysaver1024** [comment]
得，我还给自己起名叫Tragedysaver1024呢，结果我成Tragedy了，，，，，，

**Comment by Tragedysaver1024** [comment]
[![pA9W0IS.jpg](https://s21.ax1x.com/2024/08/15/pA9W0IS.jpg)](https://imgse.com/i/pA9W0IS)
到这儿了（手机相机功能坏了我从数据库上下了一张）
这地方还算挺不错的……在这里呆着也挺好啊

**Comment by qiqiang233** [comment]
这个委托我接了 不对 丸啦 我接不了啊😭 楼主多保重吧

**Comment by Hu-tang-zhu** [comment]
楼主还活着嘛<img src="https://img-static.mihoyo.com/communityweb/upload/8b198c0a467614f263e08593bd4b000d.png" alt="安比_无语" width="75" >

**Comment by Genshin-Neuvillette** [comment]
寄<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by tfuftyftyu** [comment]
为楼主祈福

**Comment by missfggc** [comment]
这边建议您继续像更深处去，这个只有很少人出来

**Comment by AetherDrop** [comment]
完辣

**Comment by missfggc** [comment]
建议去level974


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月16日(周五) 凌晨1:05
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [share121/inter-knot] [求助]我不小心走到一个有点奇怪的地方 (Discussion #572)




 
完辣
 
&mdash;

**Comment by Aitohana** [comment]
首先排除后室（）

**Comment by fulinshen** [comment]
这委托我......我接不了。

**Comment by Callkit-P0p** [comment]
这个委托~我...接个屁<img src="https://bbs-static.miyoushe.com/static/2023/05/22/a2fa805a0a782cdb8ab44ddaf8d5343d_2757305461654236072.png" alt="阿姬倒地了" width="75" >

**Comment by MrDucker** [comment]
这个委托……我接不了<img src="https://img-static.mihoyo.com/communityweb/upload/5981af706fa790734aa32abfc2df996b.png" alt="比利 柔弱" width="75" >邦布送不进去<img src="https://bbs-static.miyoushe.com/static/2023/11/16/b7b6eb7ebdc9f63ba6365e7e8d6b5fd7_5912345410510420967.png" alt="安比-啊···" width="75" >

**Comment by GanlinSama** [comment]
看楼主最新回复，大抵是……安全了？<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/d3f9322fcc499df3fca778e8c366f34d_4195564687951765835.png" alt="阿姬疑问" width="75" >

**Comment by JobNumber-9527** [comment]
祈祷自己运气好，走到前室吧，这个委托真接不了

**Comment by missfggc** [comment]
差不多，无限城市很安全


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月16日(周五) 晚上11:59
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [share121/inter-knot] [求助]我不小心走到一个有点奇怪的地方 (Discussion #572)




 
看楼主最新回复，大抵是……安全了？
 
&mdash;

**Comment by missfggc** [comment]
大底是出不来了


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月17日(周六) 凌晨0:00
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [share121/inter-knot] [求助]我不小心走到一个有点奇怪的地方 (Discussion #572)




 
祈祷自己运气好，走到前室吧，这个委托真接不了
 
&mdash;
